src/util/notifications_util.js

```diff
@@ -514,7 +514,7 @@ function compose_notification_test(req) {
 function _get_system_name(req) {
 
     if (req && req.object_sdk && req.object_sdk.nsfs_system) {
-        const name = Object.keys(req.object_sdk.nsfs_system)[0];
+        const name = process.env.NODE_NAME || os.hostname();
```

Comment:
If it is about returning the NODE name or hostname then why do we need to check this "if" condition?
We were having this "if" condition previously because we were using req.object_sdk inside "if" block.

But now, we do not need to check it.
I feel that just these two line should be enough -
        const name = process.env.NODE_NAME || os.hostname();
        return name;

Reply:
The if is to distinguish between NC and containerized (they have different handling).
I've added comments for clarification.